examples/compose/local/setup.sh

```diff
@@ -110,7 +110,12 @@ else
   password
 fi
 
-docker-compose up -d
+sudo docker-compose up -d
+
+if [[ $? != 0 ]]; then
+  echo "docker-compose encountered an error"
```

Comment:
Is this actually necessary if `docker-compose` bails out with an error it will have whatever the output of said meaningful error is presented to the user.

Reply:
I agree that could be removed.